tests/javascripts/preventDuplicateFormSubmissions.test.js

```diff
@@ -63,7 +63,7 @@ describe('Prevent duplicate form submissions', () => {
     // JSDOM doesn't implement form submissions
     // see https://github.com/jsdom/jsdom/issues/1937#issuecomment-321575590
     //
-    // If a form submission is caused by a input/button[type=submit], this instead outputs a
+    // If a form submission is caused by a input/button, this instead outputs a
```

Comment:
This, and the other replacement in this file, would read better as:
```suggestion
    // If a form submission is caused by a submit input/button, this instead outputs a
```

Also: could we update the HTML fixture under `// set up DOM` too?